config/routes.rb

```diff
@@ -1,3 +1,21 @@
+#Draw routes
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  #Set root path
+  root 'patients#index'
+
+  #Set admin path
+  get 'admin', to: 'access#menu'
+
+  #Set paths to handle admin access
+  get 'access/menu'
+  get 'access/login'
+  post 'access/attempt_login'
+  get 'access/logout'
+
+  resources :patients do  
+    member do
```

Comment:
hm.. never seen this `member` block before. Maybe all of us should read the latest edition of the Rails book!

Reply:
resources creates a route related to every CRUD action besides Delete (https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions)
So, you can add custom routes with **member** or **collection**. Member routes require an ID since they act on members. Collection routes don't, because they act on a collection of objects